keras/layers/core.py

```diff
@@ -33,6 +46,25 @@ def __init__(self, **kwargs):
         if not hasattr(self, 'params'):
             self.params = []
 
+    def __call__(self, X):
```

Comment:
I added one. An I just linked a working example. If you have some suggestions, please let me know. Also, if you have a better way and want to completely rewrite `__call__` I'm also ok with that.